podman-image/Containerfile

```diff
@@ -31,31 +27,34 @@ COPY  10-autologin.conf /etc/systemd/system/getty@.service.d/10-autologin.conf
 ## by default
 COPY delegate.conf /etc/systemd/system/user@.service.d/delegate.conf
 
-# Setup the podman-next copr repo
+# Setup the podman-next and packit copr repos.
 # The source file for the dnf repo may say `rawhide` but it's release
 # agnostic and `rawhide` in the name is unlikely to change compared
 # with URLs containing fedora release numbers.
 ADD https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman-next/repo/fedora-rawhide/rhcontainerbot-podman-next-fedora-rawhide.repos /etc/yum.repos.d/rhcontainerbot-podman-next-fedora.repo
 ADD https://download.copr.fedorainfracloud.org/results/rhcontainerbot/podman-next/pubkey.gpg /etc/pki/rpm-gpg/rhcontainerbot-podman-next-fedora.gpg
 
+ADD https://copr.fedorainfracloud.org/coprs/packit/containers-podman-${PODMAN_PR_NUM}/repo/fedora-rawhide/packit-containers-podman-${PODMAN_PR_NUM}-fedora-rawhide.repo /etc/yum.repos.d/podman-release-copr.repo
+ADD https://download.copr.fedorainfracloud.org/results/packit/containers-podman-${PODMAN_PR_NUM}/pubkey.gpg /etc/pki/rpm-gpg/podman-release-copr.gpg
+
 # 1. For dev builds, replace aardvark-dns, conmon, crun, netavark, podman, containers-common
 # 2. For release builds, fetch the build from koji
 # 3. Remove moby-engine, containerd, runc, zincati for both dev and release builds
 # Note: Currently does not result in a size reduction for the container image
 
 RUN if [[ ${PODMAN_RPM_TYPE} == "dev" ]]; then \
+        rm /etc/yum.repos.d/podman-release-copr.repo /etc/pki/rpm-gpg/podman-release-copr.gpg && \
         rpm-ostree override replace --experimental --freeze \
         --from repo="copr:copr.fedorainfracloud.org:rhcontainerbot:podman-next" \
         aardvark-dns crun netavark podman containers-common containers-common-extra crun-wasm; \
     else \
         rm /etc/yum.repos.d/rhcontainerbot*.repo /etc/pki/rpm-gpg/rhcontainerbot*.gpg && \
-        if [[ $(rpm -q podman) != "podman-${PODMAN_VERSION}-${PODMAN_RPM_RELEASE}.fc${FEDORA_RELEASE}.${ARCH}" ]]; then \
-            rpm-ostree override replace ${PODMAN_RPM_URL}; \
-        fi \
+        rpm-ostree override replace --experimental --freeze \
+        --from repo="copr:copr.fedorainfracloud.org:packit:containers-podman-${PODMAN_PR_NUM}" \
+        podman; \
     fi && \
     rpm-ostree override remove moby-engine containerd runc && \
-    rm -fr /var/cache && \
-    ostree container commit
+    rm -fr /var/cache && ostree container commit
```

Comment:
also small nit can you revert this change here, I find one command per line much easier to parse.

Reply:
done

Reply:
This is still there, same on the line below. This creates an unnecessary diff to look at.
Anyway I am not going to block over it.

Reply:
sorry about that. I misread your earlier comment. Should be fixed in the latest diff.